go.mod

```diff
@@ -98,7 +98,7 @@ require (
 	gopkg.in/yaml.v2 v2.4.0
 	k8s.io/api v0.0.0
 	k8s.io/apiextensions-apiserver v0.0.0
-	k8s.io/apimachinery v0.0.0
+	k8s.io/apimachinery v0.23.5
```

Comment:
This is the thing I was saying a rebase should fix, and needs to be reverted. Let me know if you can't get it undone after a `go mod tidy`.